lib/decompress/zstd_decompress.c

```diff
@@ -1397,6 +1403,11 @@ size_t ZSTD_DCtx_setFormat(ZSTD_DCtx* dctx, ZSTD_format_e format)
     return ZSTD_DCtx_setParameter(dctx, ZSTD_d_format, format);
 }
 
+size_t ZSTD_DCtx_setForceIgnoreChecksum(ZSTD_DCtx* dctx, ZSTD_forceIgnoreChecksum_e value)
```

Comment:
Do we need this?

I guess in my mind users would just use `ZSTD_DCtx_setParameter(..., ZSTD_d_forceIgnoreChecksum, ...)` directly.

Reply:
I presume this method may have been created due to the existence of `ZSTD_DCtx_setFormat()`.

I think the missing context here is that this capability existed before `ZSTD_DCtx_setParameter()`, so that's why it already had its own method. Furthermore, when `ZSTD_DCtx_setParameter()` started its existence, it only had _one_ parameter to look after, making it both more complex and less expressive than `ZSTD_DCtx_setFormat()`, so it was not completely obvious which one was the "redundant" one.

It's only now that `ZSTD_DCtx_setParameter()` can access multiple parameters that it starts to make more sense, and that dedicated access methods such as `ZSTD_DCtx_setFormat()` start to be seen as redundant.

Reply:
Do we want to just get rid of both then? The params they control are still technically in the "experimental" section.

Reply:
Not in this PR, which is focused on `forceIgnoreChecksum`. 
Removing a symbol, even from the experimental section, is a "dangerous" operation, as we don't always know how many other code depend on it. We may nonetheless do it, if it seems that it's beneficial for API clarity, but likely in combination with a larger version change, like `v1.5.0`.